torch/csrc/autograd/python_variable.cpp

```diff
@@ -1961,4 +1972,41 @@ c10::intrusive_ptr<TensorImpl> concrete_detach_fn(const c10::impl::PyInterpreter
   return res_t.getIntrusivePtr();
 }
 
+bool concrete_is_contiguous_fn(const c10::impl::PyInterpreter*, const c10::TensorImpl* self) {
```

Comment:
Since it looks like we're about to get a bunch of these, it would be a big help to figure out how to reduce the code duplication here. The biggest change between these functions is packing the arguments into an IValues list and then unpacking the result, everything else is generic.

Reply:
For now, I've just added a helper function between the duplicated code for `concrete_detach_fn` and `concrete_is_contiguous_fn`